src/com/amazon/ion/benchmark/GeneratorOptions.java

```diff
@@ -4,11 +4,13 @@
 
 import java.util.List;
 import java.util.Map;
+import java.util.Random;
 
 /**
  * Execute Ion Data Generator after receiving the hashmap of command line options.
  */
 public class GeneratorOptions {
+    public static Random randomSeed;
```

Comment:
`randomSeed` doesn't feel like the right name for this. This is just a `Random` instance that may or may not have been seeded. `random` is probably a fine name.